WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -225,11 +223,15 @@ public void onResume() {
     }
 
     @Override
-    protected void onSaveInstanceState(Bundle outState) {
+    protected void onSaveInstanceState(@NonNull Bundle outState) {
         outState.putInt(KEY_SITE_LOCAL_ID, mCurrentLocalId);
         outState.putBoolean(KEY_IS_IN_SEARCH_MODE, getAdapter().getIsInSearchMode());
         outState.putString(KEY_LAST_SEARCH, getAdapter().getLastSearch());
-        outState.putBoolean(KEY_REFRESHING, mSwipeToRefreshHelper.isRefreshing());
+        if (mSwipeToRefreshHelper != null) {
+            outState.putBoolean(KEY_REFRESHING, mSwipeToRefreshHelper.isRefreshing());
+        } else {
+            outState.putBoolean(KEY_REFRESHING, false);
+        }
```

Comment:
🔍  Nitpick: this is completely minor, but since `KEY_REFRESHING` is always populated, maybe using some boolean logic would make this shorter?

```suggestion
        outState.putBoolean(KEY_REFRESHING, mSwipeToRefreshHelper != null && mSwipeToRefreshHelper.isRefreshing());
```